hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,61 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ "$platform" == "windows/amd64" ]]; then
+        os::build::version::get_vars
```

Comment:
I will take a look tomorrow if you guys don't have bandwidth digging that, also about that add `+` if patch exist @fabianofranz @stevekuznetsov  wdyt ?